skimage/restoration/_denoise.py

```diff
@@ -10,39 +9,89 @@
 import numbers
 
 
-def _gaussian_weight(sigma_sqr, value, *, dtype=float):
-    return np.exp(-0.5 * value * value / sigma_sqr, dtype=dtype)
+def _gaussian_weight(array, sigma_squared, *, dtype=float):
+    """Helping function. Define a Gaussian weighting from array and
+    sigma_square.
+
+    Parameters:
+    -----------
+    array : ndarray
+        Input array.
+    sigma_squared : float
+        The squared standard deviation used in the filter.
+    dtype : data type object
+        The type and size of the data to be returned.
+
+    Returns:
+    --------
+    gaussian : ndarray
+        The input array filtered by the Gaussian.
+    """
+    return np.exp(-0.5 * (array ** 2  / sigma_squared), dtype=dtype)
 
 
 def _compute_color_lut(bins, sigma, max_value, *, dtype=float):
-    values = np.linspace(0, max_value / bins, bins, endpoint=False)
-    color_lut = _gaussian_weight(sigma**2, values, dtype=dtype)
-    return color_lut
+    """Helping function. Define a lookup table containing Gaussian filter
+    values using the color distance sigma.
+
+    Parameters:
```

Comment:
nit-picky comments ON.
```suggestion
    Parameters
```
And below... Don't forget to shorten the dashes.

Reply:
I mixed the style of our docs with something I was typing here these days :sweat_smile: Sorry about that!